lib/install/.circleci/config.yml

```diff
@@ -201,15 +172,10 @@ jobs:
 
     steps:
       - checkout
+      - ruby/install-deps:
+          with-cache: false
```

Comment:
why we disable cache here? maybe we could change cache key?

Reply:
Two reasons: 
- we didn't use cache here initially;
- we could change cache key here, but in orb it'll use `Gemfile.lock` file for cache key, instead of `Gemfile.tools.lock`as stated in the [docs](https://circleci.com/orbs/registry/orb/circleci/ruby#commands-install-deps):
![Screenshot from 2020-05-21 13-44-03](https://user-images.githubusercontent.com/1931210/82551757-7b41c280-9b69-11ea-8578-f3cee902c0da.png)

Reply:
let's no use orb for tools then

Reply:
but we need to have cache for tools because this is very expensive operation

Reply:
@pftg `bundle install` is not performed in `tools_upgrade` job, sorry for the confusion. Restored bundler install step, and added `bundler-version` parameter to `install-deps` ruby orb command.